XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -291,6 +299,17 @@ private void Animated_ScrollViewer_ViewChanging(object sender, ScrollViewerViewC
             }
         }
 
+        private void SetUIANamesForSelectedEntry(Button selectedItem)
+        {
+            if (LastSelectedColorButton != null && LastSelectedColorButton.Content is string content)
+            {
+                AutomationProperties.SetName(LastSelectedColorButton, content);
+            }
+
+            AutomationProperties.SetName(selectedItem, (string)selectedItem.Content + " , selected");
```

Comment:
Do we need a null check for selected item?

Reply:
You could but the code would fail on different areas if the passed parameter would be null so there is no benefit in adding that I think.